src/arviz_plots/backend/matplotlib/__init__.py

```diff
@@ -119,6 +119,37 @@ def get_default_aes(aes_key, n, kwargs=None):
     return get_agnostic_default_aes(aes_key, n, kwargs)
 
 
+def dealiase_aes_value(aes, value):
+    try:
+        if value.startswith(f"{aes}_"):
+            index = int(value.rsplit("_")[-1])
+        else:
+            index = int(value[1:])
+        dealiased_value = get_default_aes(aes, index+1)[index]
+    except ValueError:
+        return value
+    return dealiased_value
+
+
+def expand_aesthetic_aliases(plot_fn):
+    def _dealiased_plot_fn(*args, **kwargs):
+        for aes in ("color", "facecolor", "edgecolor"):
+            if (
+                aes in kwargs 
+                and isinstance(value := kwargs[aes], str)
+                and (value[0].lower() in ("c", aes[0]) or value.startswith(f"{aes}_"))
+            ):
```

Comment:
allow only `C0`, `L2`... no `color_0`

Reply:
Now only "CX" syntax is allowed. I think it's fine to use only "CX" instead of CX, LX, MX. With "C" being from "cycler", not color.